production/scrapers/delaware.R

```diff
@@ -12,26 +12,27 @@ delaware_check_date <- function(x, date = Sys.Date()){
 delaware_pull <- function(x){
     "1VhAAbzipvheVRG0UWKMLT6mCVQRMdV98lUUkk-PCYtQ" %>%
         googlesheets4::read_sheet(sheet = "DE", 
-                                  col_types = "Dccccccccccc")
+                                  col_types = "Dcccccccccccc")
 }
 
 delaware_restruct <- function(x){
     x %>%
         filter(!is.na(Date)) %>% 
-        filter(Date == max(Date))
+        filter(Date == max(Date)) 
```

Comment:
If you delete the commented out line, perhaps you could remove this extra space too. To me, it's better to limit our changes to intentional ones. But I won't let this hold up the PR.